### ISSUE TITLE
[BITV] 9.4.1.2/6.3 - "Change scope" menus - Alle menus have been implemented using the aria-haspopup="menu" attribute, which informs users that a menu is being opened.(1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41903]
> [!IMPORTANT]
> For QA, please just check that PR is merged to stable28

 However, even though the menu items can be navigated using arrow keys, the corresponding ARIA roles role="menu" and role="menuitem" are completely missing. 

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v6-n3

**Comment by JuliaKirschenheuter** [comment]
@ShGKme has this issue been resolved after call with Michael?

**Comment by ShGKme** [comment]
> @ShGKme has this issue been resolved after call with Michael?

No, this menu is a bit special with a custom design. I'll add more later

**Comment by JuliaKirschenheuter** [comment]
Ok, thank you! Moved issue to "Backlog"

**Comment by ShGKme** [comment]
Pending backport

**Comment by JuliaKirschenheuter** [comment]
closing as done

**Comment by JuliaKirschenheuter** [comment]
closing as done

**Comment by szaimen** [comment]
I tested this and after closing the popup, the focus is lost to body:
![image](https://github.com/nextcloud/server/assets/42591237/02e210b9-eab6-496c-b922-972e1289463b)

**Comment by JuliaKirschenheuter** [comment]
> I tested this and after closing the popup, the focus is lost to body: ![image](https://private-user-

Fix: https://github.com/nextcloud/server/pull/43358
